types/global.d.ts

```diff
@@ -23,7 +23,7 @@ declare class Platform {
 
   closeCurrentWindow: () => void;
 
-  openExtensionInBrowser?: (_1, _1, condition: boolean) => void;
+  openExtensionInBrowser?: (_1, _1?, condition?: boolean) => void;
```

Comment:
It was required since I've converted a `.js` file into a `.ts` file. This file was using it with 1 parameter, so I didn't wanted to change the call itself (since it was working initially).

So I figured that this declaration was probably "wrong".

Do you have any recommendation here?